api/src/pcapi/core/educational/api.py

```diff
@@ -1183,6 +1183,16 @@ def synchronize_adage_ids_on_venues() -> None:
 
         venue.adageId = str(filtered_cultural_partner_by_ids[venue.id].id)
 
+    offerers: list[offerers_models.Offerer] = (
+        offerers_models.Offerer.query.join(offerers_models.Venue, offerers_models.Offerer.managedVenues)
+        .filter(offerers_models.Venue.id.in_(filtered_cultural_partner_by_ids.keys()))
+        .all()
+    )
+
+    for offerer in offerers:
+        if offerer.isForbiddenForEAC:
+            offerer.isForbiddenForEAC = False
+
```

Comment:
En une seule requête SQL : 

```suggestion
    offerers = (
        offerers_models.Offerer.query.filter(isForbiddenForEAC.is_(True))
         # astuce ci-dessous : pas besoin de fournir le modèle et la relation. La relation suffit.
        .join(offerers_models.Offerer.managedVenues)
        .filter(offerers_models.Venue.id.in_(filtered_cultural_partner_by_ids.keys()))
    )
    offerers.update({"isForbiddenForEAC": False}, synchronize_session=False)
```

Reply:
On m'a dit que c'était toujours mieux de fournir la relation sur laquelle faire le join (plus clair et peut éviter des bugs si on a plusieurs jointures sur la même table)
Mais sinon oui effectivement je prends l'ajout du filtre sur isForbiddenForEAC :D

Reply:
> On m'a dit que c'était toujours mieux de fournir la relation sur laquelle faire le join

C'est ce que je propose : préciser le nom de la relation (ici `Offerer.managedVenues`) sans préciser le nom du modèle "cible" de la relation (ici `Venue`).

Reply:
Ah oui non pardon j'ai lu tout de travers !!